ui/lib/sync/addon/components/secrets/page/overview.ts

```diff
@@ -52,6 +57,24 @@ export default class SyncSecretsDestinationsPageComponent extends Component<Args
     return this.args.activatedFeatures.includes('secrets-sync');
   }
 
+  get adapterErrorMessage() {
```

Comment:
We won't have an adapterError from getting this endpoint anymore, we can remove!